doc/radosgw/layout.rst

```diff
@@ -178,22 +183,22 @@ Known pools:
   prodtx$prodt.buckets
   test2
 
-.users.email
+  namespace: users.email
```

Comment:
the formatting doesn't turn out very well here, this is how it renders:
> namespace: users.email Unimportant

maybe try adding another level of indentation under the namespaces? ie
```rst
<zone>.rgw.meta
  ...
  namespace: users.email
    Unimportant
```